.gitlab/pipelines/stages/deploy/deploy_cf_operator.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+# shellcheck disable=SC1090
+source "$(bazel info workspace)/.gitlab/pipelines/config/config.sh"
+
+# Install cf-operator.
+chart="$(mktemp -t cf_operator_XXXXXXXX.tgz)"
+curl -L -o "${chart}" "${CF_OPERATOR_CHART_URL}"
+bazel run @helm//:helm -- template "${chart}" --name cf-operator --namespace "${KUBECF_NAMESPACE}" | bazel run @kubectl//:kubectl -- apply -f - --namespace "${KUBECF_NAMESPACE}"
```

Comment:
This sounds like it would make sense as a bazel command (as in, `bazel run @operator//:deploy` or something).

Reply:
Indeed. I'll do the move of cf-operator to defs.bzl in a separate PR, then it'll be more appropriate to have a command like this.